ludwig/features/base_feature.py

```diff
@@ -220,7 +215,7 @@ def update_metrics(self, targets, predictions):
                 metric_fn.update(predictions, targets)
             else:
                 #metric_fn.update_state(targets, predictions[PREDICTIONS])
-                metric_fn.update(predictions[PREDICTIONS], targets)
+                metric_fn.update(predictions, targets)
```

Comment:
This is in the base feature, so should work across the baord for all features and all metrics

Reply:
Resolved offline